library/spdm_responder_lib/libspdm_rsp_encap_get_certificate.c

```diff
@@ -194,7 +198,7 @@ return_status libspdm_process_encap_response_certificate(
                 &spdm_context->encap_context.certificate_chain_buffer),
             libspdm_get_managed_buffer_size(
                 &spdm_context->encap_context.certificate_chain_buffer),
-            NULL, NULL);
+            NULL, NULL, is_requester);
```

Comment:
Can use a `false` literal and remove `is_requester` variable.

Reply:
Thanks. I have changed the code to `false`.